backend/package.json

```diff
@@ -7,6 +7,7 @@
     "dev": "cargo run",
     "lint": "cargo fmt -- --check && cargo clippy",
     "lint:fix": "cargo fmt && cargo clippy --fix",
+    "lint:fix:unsafe": "cargo fmt && cargo clippy --fix --allow-staged --allow-dirty",
```

Comment:
I think we could just merge that with `lint:fix`. The point of this script is to reformat code. Eslint just doesn't care about status of those files, so we can do the same for clippy ig.